x11-packages/emacs-x/lisp-textmodes-ispell.el.patch.beforehostbuild

```diff
@@ -21,6 +20,6 @@ diff -u -r ../emacs-26.2/lisp/textmodes/ispell.el ./lisp/textmodes/ispell.el
 +	((file-readable-p "@TERMUX_PREFIX@/share/lib/dict/words")
```

Comment:
Could you fix the lines above in the patch while you are at? Seems this patch has been half-broken since emacs was added, should be:
```
  (cond ((file-readable-p "@TERMUX_PREFIX@/dict/web2") "@TERMUX_PREFIX@/dict/web2")
      ((file-readable-p "@TERMUX_PREFIX@/share/dict/web2") "@TERMUX_PREFIX@/share/dict/web2")
      ((file-readable-p "@TERMUX_PREFIX@/dict/words") "@TERMUX_PREFIX@/dict/words")
      ((file-readable-p "@TERMUX_PREFIX@/lib/dict/words") "@TERMUX_PREFIX@/lib/dict/words")
      ((file-readable-p "@TERMUX_PREFIX@/share/dict/words") "@TERMUX_PREFIX@/share/dict/words")
```